oop/oop.js

```diff
@@ -0,0 +1,61 @@
+class Character {
+    static charactersCount = 0;
+
+    constructor(){
+        // prevents instantiation
+        if (new.target === Character){
```

Comment:
You can also say `if (this.constructor === Character)`, it looks more understandable